src/tokens/ERC1155/presets/pack/IERC1155Pack.sol

```diff
@@ -42,7 +42,7 @@ interface IERC1155Pack {
     event Commit(address indexed user, uint256 blockNumber);
 
     /// @notice Emitted when a reveal is successful
-    event Reveal(address user, PackContent packContent);
```

Comment:
I don't think we want indexed here. It's extra gas and our indexer is capable of filtering all parameters.